openedx/features/badging/tests/test_helpers.py

```diff
@@ -279,3 +275,17 @@ def test_filter_earned_badge_by_joined_team_course_with_no_team(self):
 
         self.assertIsNone(course_team)
         self.assertEqual(earned_badges, list())
+
+    @mock.patch('openedx.features.badging.helpers.publish_notification_to_user')
+    def test_send_user_badge_notification(self, publish_notification_to_user):
```

Comment:
Lets rename `publish_notification_to_user` to `mock_publish_notification_to_user`

Reply:
done